src/Oro/Bundle/UserBundle/Migrations/Schema/v1_9/OroUserBundle.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Oro\Bundle\UserBundle\Migrations\Schema\v1_9;
```

Comment:
Please change migration version because in nearest release we have migration v 1_9